src/main/java/ch/njol/skript/classes/AnyInfo.java

```diff
@@ -0,0 +1,16 @@
+package ch.njol.skript.classes;
+
+import ch.njol.skript.lang.util.common.AnyProvider;
+
+public class AnyInfo<Type extends AnyProvider> extends ClassInfo<Type> {
+
+	/**
+	 * @param c        The class
+	 * @param codeName The name used in patterns
+	 */
+	public AnyInfo(Class<Type> c, String codeName) {
+		super(c, "any-" + codeName);
```

Comment:
I wanted some way to differentiate it from regular classinfos so that it's a bit more recognizable in patterns (e.g. `name of %player/entity/any-named%`) and make sure it can't conflict with existing things.